ios/Classes/TiWebdialogAuthenticationSessionProxy.m

```diff
@@ -7,45 +7,85 @@
  */
 
 #if IS_IOS_11
-
-#import "TiWebdialogAuthenticationSessionProxy.h"
+#if IS_IOS_12
+#import <AuthenticationServices/AuthenticationServices.h>
+#endif
 #import "TiUtils.h"
+#import "TiWebdialogAuthenticationSessionProxy.h"
+#import <SafariServices/SafariServices.h>
 
 @implementation TiWebdialogAuthenticationSessionProxy
 
-- (SFAuthenticationSession *)authSession
+- (id)authSession
 {
   if (_authSession == nil) {
     NSString *url = [TiUtils stringValue:[self valueForKey:@"url"]];
     NSString *scheme = [TiUtils stringValue:[self valueForKey:@"scheme"]];
 
-    _authSession = [[SFAuthenticationSession alloc] initWithURL:[TiUtils toURL:url proxy:self]
-                                              callbackURLScheme:[TiUtils stringValue:scheme]
-                                              completionHandler:^(NSURL *callbackURL, NSError *error) {
-                                                NSMutableDictionary *event = [NSMutableDictionary dictionaryWithDictionary:@{
-                                                  @"success" : NUMBOOL(error == nil)
+#if IS_IOS_12
```

Comment:
This is not required, since Xcode 10 is minimum when developing iOS apps.

Reply:
Removed. Thanks!